### PR TITLE
Remove reference to archived repo

### LINKED ISSUE #738: Page contains link to archived repository
https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure#tips-for-working-with-terraform contains a link to an archived repository: https://github.com/rancher/terraform-modules

### PR DESCRIPTION
**Pull request by btat**
Fixes #738 

## Description

This PR removes the bullet point referencing the archived repository.

### CODE REVIEW

**Review by sunilarjun — APPROVED** [review]

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure.md [hunk 1]
```diff
@@ -50,11 +50,7 @@ When you need to make changes to your infrastructure, instead of manually updati
 
 - You can also modify auth in the Terraform provider.
 
-- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.
-
-- If you want to manage Kubernetes cluster settings, Rancher settings, and hardware settings all in one place, use [Terraform modules](https://github.com/rancher/terraform-modules). You can pass a cluster configuration YAML file or an RKE template configuration file to a Terraform module so that the Terraform module will create it. In that case, you could use your infrastructure-as-code to manage the version control and revision history of both your Kubernetes cluster and its underlying hardware.
-
-## Tip for Creating CIS Benchmark Compliant Clusters
+- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.## Tip for Creating CIS Benchmark Compliant Clusters
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure.md:53]:
```suggestion
- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.

## Tip for Creating CIS Benchmark Compliant Clusters
```

File: versioned_docs/version-2.0-2.4/how-to-guides/advanced-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure.md [hunk 2]
```diff
@@ -50,11 +50,7 @@ When you need to make changes to your infrastructure, instead of manually updati
 
 - You can also modify auth in the Terraform provider.
 
-- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.
-
-- If you want to manage Kubernetes cluster settings, Rancher settings, and hardware settings all in one place, use [Terraform modules](https://github.com/rancher/terraform-modules). You can pass a cluster configuration YAML file or an RKE template configuration file to a Terraform module so that the Terraform module will create it. In that case, you could use your infrastructure-as-code to manage the version control and revision history of both your Kubernetes cluster and its underlying hardware.
-
-## Tip for Creating CIS Benchmark Compliant Clusters
+- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.## Tip for Creating CIS Benchmark Compliant Clusters
```
Comment (martyav) [inline versioned_docs/version-2.0-2.4/how-to-guides/advanced-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure.md:53]:
```suggestion
- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.

## Tip for Creating CIS Benchmark Compliant Clusters
```

File: versioned_docs/version-2.5/how-to-guides/advanced-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure.md [hunk 3]
```diff
@@ -50,11 +50,7 @@ When you need to make changes to your infrastructure, instead of manually updati
 
 - You can also modify auth in the Terraform provider.
 
-- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.
-
-- If you want to manage Kubernetes cluster settings, Rancher settings, and hardware settings all in one place, use [Terraform modules](https://github.com/rancher/terraform-modules). You can pass a cluster configuration YAML file or an RKE template configuration file to a Terraform module so that the Terraform module will create it. In that case, you could use your infrastructure-as-code to manage the version control and revision history of both your Kubernetes cluster and its underlying hardware.
-
-## Tip for Creating CIS Benchmark Compliant Clusters
+- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.## Tip for Creating CIS Benchmark Compliant Clusters
```
Comment (martyav) [inline versioned_docs/version-2.5/how-to-guides/advanced-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure.md:53]:
```suggestion
- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.

## Tip for Creating CIS Benchmark Compliant Clusters
```

File: versioned_docs/version-2.6/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure.md [hunk 4]
```diff
@@ -50,11 +50,7 @@ When you need to make changes to your infrastructure, instead of manually updati
 
 - You can also modify auth in the Terraform provider.
 
-- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.
-
-- If you want to manage Kubernetes cluster settings, Rancher settings, and hardware settings all in one place, use [Terraform modules](https://github.com/rancher/terraform-modules). You can pass a cluster configuration YAML file or an RKE template configuration file to a Terraform module so that the Terraform module will create it. In that case, you could use your infrastructure-as-code to manage the version control and revision history of both your Kubernetes cluster and its underlying hardware.
-
-## Tip for Creating CIS Benchmark Compliant Clusters
+- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.## Tip for Creating CIS Benchmark Compliant Clusters
```
Comment (martyav) [inline versioned_docs/version-2.6/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure.md:53]:
```suggestion
- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.

## Tip for Creating CIS Benchmark Compliant Clusters
```

File: versioned_docs/version-2.7/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure.md [hunk 5]
```diff
@@ -50,11 +50,7 @@ When you need to make changes to your infrastructure, instead of manually updati
 
 - You can also modify auth in the Terraform provider.
 
-- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.
-
-- If you want to manage Kubernetes cluster settings, Rancher settings, and hardware settings all in one place, use [Terraform modules](https://github.com/rancher/terraform-modules). You can pass a cluster configuration YAML file or an RKE template configuration file to a Terraform module so that the Terraform module will create it. In that case, you could use your infrastructure-as-code to manage the version control and revision history of both your Kubernetes cluster and its underlying hardware.
-
-## Tip for Creating CIS Benchmark Compliant Clusters
+- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.## Tip for Creating CIS Benchmark Compliant Clusters
```
Comment (martyav) [inline versioned_docs/version-2.7/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure.md:53]:
```suggestion
- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.

## Tip for Creating CIS Benchmark Compliant Clusters
```

File: versioned_docs/version-2.8/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure.md [hunk 6]
```diff
@@ -50,11 +50,7 @@ When you need to make changes to your infrastructure, instead of manually updati
 
 - You can also modify auth in the Terraform provider.
 
-- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.
-
-- If you want to manage Kubernetes cluster settings, Rancher settings, and hardware settings all in one place, use [Terraform modules](https://github.com/rancher/terraform-modules). You can pass a cluster configuration YAML file or an RKE template configuration file to a Terraform module so that the Terraform module will create it. In that case, you could use your infrastructure-as-code to manage the version control and revision history of both your Kubernetes cluster and its underlying hardware.
-
-## Tip for Creating CIS Benchmark Compliant Clusters
+- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.## Tip for Creating CIS Benchmark Compliant Clusters
```
Comment (martyav) [inline versioned_docs/version-2.8/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/infrastructure.md:53]:
```suggestion
- You can reverse engineer how to do define a setting in Terraform by changing the setting in Rancher, then going back and checking your Terraform state file to see how it maps to the current state of your infrastructure.

## Tip for Creating CIS Benchmark Compliant Clusters
```

**Review by martyav — CHANGES_REQUESTED** [review]
Small typo in spacing that broke a heading

**Review by martyav — APPROVED** [review]

**Pull request merged**